app/request/api/views.py

```diff
@@ -88,10 +88,10 @@ def edit_request_info():
         previous_value['title'] = current_request.title
         new_value['title'] = val
         type_ = event_type.REQ_TITLE_EDITED
-    elif edit_request['name'] == 'agency_description':
-        previous_value['agency_description'] = current_request.agency_description
-        new_value['agency_description'] = val
-        type_ = event_type.REQ_AGENCY_DESC_EDITED
+    elif edit_request['name'] == 'agency_request_summary':
```

Comment:
When editing the agency request summary, this if statement is not executed because the name being passed in from the front is `agency-request-summary`. Change this to check to be ` == 'agency-request-summary'` or change the id to be `agency_request_summary`.